src/lib/journey/callbacks/confirmation/confirmation.mock.ts

```diff
@@ -66,6 +66,37 @@ export default {
         },
       ],
     },
+    {
+      type: CallbackType.ConfirmationCallback,
+      output: [
+        {
+          name: 'prompt',
+          value: '',
+        },
+        {
+          name: 'messageType',
+          value: 0,
+        },
+        {
+          name: 'options',
+          value: ['I confirm', 'No, I do not confirm'],
+        },
+        {
+          name: 'optionType',
+          value: -1,
```

Comment:
It doesn't seem to affect what we're doing here, but out of interest do we know what this means?